adiwg-mdtranslator.gemspec

```diff
@@ -39,6 +39,6 @@ Gem::Specification.new do |spec|
    spec.add_runtime_dependency "jbuilder", "~> 2.5"
    spec.add_runtime_dependency "kramdown", ">= 1.13", "< 3.0"
    spec.add_runtime_dependency "coderay", "~> 1.1"
-   spec.add_runtime_dependency "nokogiri", "~> 1.7"
+   spec.add_runtime_dependency "nokogiri", "~> 1.15"
```

Comment:
Did this need to be changed? We might need to do a more complete update/upgrade, but unless this is absolutely necessary we shouldn't upgrade this here.